common/src/main/scala/uk/ac/wellcome/models/UnifiedItem.scala

```diff
@@ -3,12 +3,16 @@ package uk.ac.wellcome.models
 import com.sksamuel.elastic4s._
 import uk.ac.wellcome.utils.JsonUtil
 
-case class Identifier(source: String, sourceId: String, value: String)
+case class Identifier(CanonicalID: String, MiroID: String)
+
+case class IdentifiedUnifiedItem(canonicalId: String, unifiedItem: UnifiedItem)
+
+case class SourceIdentifier(source: String, sourceId: String, value: String)
```

Comment:
Perhaps add a couple of Scaladoc comments for additional clarity? Something like:

```scala
/** A set of identifiers as stored in DynamoDB */
case class Identifier

/** An identifier received from one of the original sources */
case class SourceIdentifier

/** A representation of an item in our ontology, without a canonical identifier */
case class UnifiedItem
```

Just a suggestion.